src/productcatalogservice/server_test.go

```diff
@@ -29,7 +29,7 @@ import (
 
 func TestServer(t *testing.T) {
 	ctx := context.Background()
-	addr := run(0)
+	addr := run("8080")
```

Comment:
In `master` there is already a commit that fixes that broken test case :)
We could try to rebase this branch on `master`